web/src/app/store.ts

```diff
@@ -14,16 +14,6 @@ const store = configureStore({
     getDefaultMiddleware({ immutableCheck: false, serializableCheck: false }).concat(thunkMiddleware)
 })
 
-if (process.env.NODE_ENV === 'development' && module.hot) {
```

Comment:
Do you know if we get the equivalent behaviour with vite out of the box? Or will we have to re-start our debugging session if we are making changes to a reducer?

Reply:
hmmm. I feel like vite should handle this well since a big focus of it is not having to rebundle stuff https://vitejs.dev/guide/why.html#why-vite . But i haven't done much reducer stuff lately. Can you break down for me what problem this was solving? you don't want to lose some state in redux?

Reply:
I was curious more than anything so not a big deal. I think you are correct in that it prevented loss of state in redux. I don't think we need to do anything or look into this any further. Thanks!